.github/workflows/test.yml

```diff
@@ -85,3 +85,18 @@ jobs:
           TF_ACC: "1"
         run: go test -v -cover ./...
         timeout-minutes: 10
+  e2e-tests:
+    name: Integration Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+      - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+      - uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 # v2.0.3
+        with:
+          terraform_version: ${{ matrix.terraform }}
+          terraform_wrapper: false
+      - name: Run tests
+        run: go run tests/main.go
+        env:
+          APONO_PERSONAL_TOKEN: ${{ secrets.APONO_PERSONAL_TOKEN }}
```

Comment:
lets give some more expicit name to the secret
like TERRAFORM_TEST_ACCOUNT_PERSONAL_TOKEN